### DISCUSSION TITLE
@formily/vue  onFormValuesChange 监听不到form表单值的变化

### DISCUSSION THREAD

**Discussion by yangshuwei** [alibaba/formily#1613] [Q&A 2.x] [answered]
不知道是我写的问题还是其他问题，之前可以监听到 麻烦帮看一下[https://codesandbox.io/s/dreamy-antonelli-nr0c5?file=/src/App.vue](https://codesandbox.io/s/dreamy-antonelli-nr0c5?file=/src/App.vue)

**Comment by MisicDemone** [comment, marked as answer]
与这个issue同一个原因，已修复，下版本更新：https://github.com/alibaba/formily/issues/1609
